web-app/app/tricorder/Configure.tsx

```diff
@@ -1,52 +1,53 @@
-import { useEffect } from "react";
-import { buttonClass } from "./classes";
+export type Config = {
+  antimatter?: boolean;
+  slingshot?: boolean;
+  stembolts?: boolean;
+};
 
-export function Configure(props: any) {
-  const onChange = props.onChange;
-  const configuration: Record<any, any> = {};
-  useEffect(() => {
-    localStorage.setItem(
-      "__tricorder__configuration",
-      JSON.stringify(configuration)
-    );
-  }, [configuration]);
-  useEffect(() => {
-    const newConfiguration = JSON.parse(
-      localStorage.getItem("__tricorder_configuration") ?? "{}"
-    );
-    Object.keys(newConfiguration).map((configKey) => {
-      configuration[configKey] = newConfiguration[configKey];
-    });
-  }, []);
+export function Configure({
```

Comment:
We're specifying props for Configure now! This is important as it helps us know how to implement the component.